db/view/view_building_worker.cc

```diff
@@ -244,7 +244,7 @@ future<> view_building_worker::create_staging_sstable_tasks() {
 
     // Move staging sstables from `_sstables_to_register` (on shard0) to `_staging_sstables` on corresponding shards.
     // Firstly reorgenize `_sstables_to_register` for easier movement.
-    // This is done in separate loop after commiting the group0 command, because we need to move values from `_sstables_to_register`
+    // This is done in separate loop after committing the group0 command, because we need to move values from `_sstables_to_register`
```

Comment:
```suggestion
    // This is done in a separate loop after committing the group0 command, because we need to move values from `_sstables_to_register`
```